CreeDictionary/API/models.py

```diff
@@ -356,10 +361,57 @@ def fetch_lemma_by_user_query(user_query: str, **kwargs) -> "CreeAndEnglish":
         user_query = user_query.lower()
 
         # build up result_lemmas in 2 ways
-        # 1. spell relax in descriptive fst
+        # 1. affix search (return all results that ends/starts with the query string)
+        # 2. spell relax in descriptive fst
         # 2. definition containment of the query word
+
         cree_results: Set[CreeResult] = set()
 
+        if (
+            len(user_query) > 4
+        ):  # there will be too many matches for some shorter queries
```

Comment:
It's SUPER WEIRD that Black chose to format it this way... what happens if you get rid of the outer parens?

Reply:
black insists to format this way and adds the parenthesis back.

That's cuz the comment is on the same line and black doesn't like any line exceding the maximum length. I moved the comment up a line and it looks a lot better.

Reply:
Ah! I did not know that about Black. I tend to put this kind of comment above the `if`, so maybe that's why I've never encountered it!